### PR TITLE
Table headers are now FirstLetter case

### PR DESCRIPTION
**Pull request by wallyworld**
All tables headers are now FirstLetter case. Except for things like "Model UUID" because UUID is generally capitalised.

QA - bootstrap and run various list related commands.

### CODE REVIEW

File: cmd/juju/machine/list_test.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func (s *MachineListCommandSuite) TestMachine(c *gc.C) {
 	context, err := testing.RunCommand(c, newMachineListCommand())
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(testing.Stdout(context), gc.Equals, ""+
-		"MACHINE  STATE    DNS       INS-ID               SERIES  AZ\n"+
+		"Machine  State    DNS       Inst Id              Series  AZ\n"+
```
Comment (anastasiamac) [inline cmd/juju/machine/list_test.go]: This should be either "Inst ID" as with "Model UUID" columns or "Inst id" as with "Last connection"

File: cmd/juju/status/output_tabular.go [hunk 2]
```diff
@@ -240,7 +240,7 @@ func getModelMessage(model modelStatus) string {
 
 func printMachines(tw *ansiterm.TabWriter, machines map[string]machineStatus) {
 	w := output.Wrapper{tw}
-	w.Println("MACHINE", "STATE", "DNS", "INS-ID", "SERIES", "AZ")
+	w.Println("Machine", "State", "DNS", "Inst Id", "Series", "AZ")
```
Comment (anastasiamac) [inline cmd/juju/status/output_tabular.go]: same

File: cmd/juju/storage/filesystemlist_test.go [hunk 3]
```diff
@@ -88,7 +88,7 @@ func (s *ListSuite) TestFilesystemListWithErrorResults(c *gc.C) {
 }
 
 var expectedFilesystemListTabular = `
-MACHINE  UNIT         STORAGE      ID   VOLUME  PROVIDER-ID                       MOUNTPOINT  SIZE    STATE      MESSAGE
+Machine  Unit         Storage      Id   Volume  Provider Id                       Mountpoint  Size    State      Message
```
Comment (anastasiamac) [inline cmd/juju/storage/filesystemlist_test.go]: ditto

File: cmd/plugins/juju-metadata/listformatter.go [hunk 4]
```diff
@@ -27,7 +27,7 @@ func formatMetadataTabular(writer io.Writer, metadata []MetadataInfo) {
 	print := func(values ...string) {
 		fmt.Fprintln(tw, strings.Join(values, "\t"))
 	}
-	print("SOURCE", "SERIES", "ARCH", "REGION", "IMAGE-ID", "STREAM", "VIRT-TYPE", "STORAGE-TYPE")
+	print("Source", "Series", "Arch", "Region", "Image Id", "Stream", "Virt Type", "Storage Type")
```
Comment (anastasiamac) [inline cmd/plugins/juju-metadata/listformatter.go]: and here

File: resource/cmd/output_tabular_test.go [hunk 5]
```diff
@@ -252,14 +252,14 @@ func (s *SvcTabularSuite) TestFormatSvcTabularMulti(c *gc.C) {
 	// Notes: sorted by name, then by revision, newest first.
 	c.Check(data, gc.Equals, `
 [Service]
-RESOURCE  SUPPLIED BY  REVISION
+Resource  Supplied By  Revision
```
Comment (anastasiamac) [inline resource/cmd/output_tabular_test.go]: Why is "by" capitalised but "connection" in "Last connection" isn't?

Comment (wallyworld) [inline resource/cmd/output_tabular_test.go]: I missed that one, fixed

File: cmd/juju/machine/list_test.go (hunk 1 above)
Comment (wallyworld) [inline cmd/juju/machine/list_test.go]: Fair point, I thought Id looked better. I've changed to id. We can also tweak later.

File: cmd/juju/status/output_tabular.go (hunk 2 above)
Comment (wallyworld) [inline cmd/juju/status/output_tabular.go]: fixed

File: cmd/juju/storage/filesystemlist_test.go (hunk 3 above)
Comment (wallyworld) [inline cmd/juju/storage/filesystemlist_test.go]: fixed

File: cmd/plugins/juju-metadata/listformatter.go (hunk 4 above)
Comment (wallyworld) [inline cmd/plugins/juju-metadata/listformatter.go]: fixed

File: payload/status/output_tabular_test.go [hunk 6]
```diff
@@ -75,7 +75,7 @@ func (s *outputTabularSuite) TestFormatTabularMulti(c *gc.C) {
 
 	c.Check(buff.String(), gc.Equals, `
 [Unit Payloads]
-UNIT                   MACHINE  PAYLOAD-CLASS  STATUS   TYPE    ID       TAGS         
+Unit                   Machine  Payload Class  Status   Type    Id       Tags
```
Comment (anastasiamac) [inline payload/status/output_tabular_test.go]: Sorry, should this one be a "Payload class" too?

Comment (wallyworld) [inline payload/status/output_tabular_test.go]: doh. fixed

**Comment by wallyworld** [comment]
$$merge$$

**Pull request merged**